### PR TITLE
Arsenal - Add Night Vision support stat for scopes

### PR DESCRIPTION
**Pull request by dedmen**

### CODE REVIEW

File: addons/arsenal/functions/fnc_statTextStatement_scopeVisionMode.sqf [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ * Return Value:
+ * String to display
+ *
+ * Public: No
+ */
+
+params ["", "_config"];
+TRACE_1("statTextStatement_scopeVisionMode",_config);
+
+private _opticsModes = ("true" ConfigClasses (_config >> "ItemInfo" >> "OpticsModes")) apply {
```
Comment (Dystopian) [inline addons/arsenal/functions/fnc_statTextStatement_scopeVisionMode.sqf]:
```suggestion
private _opticsModes = ("true" configClasses (_config >> "ItemInfo" >> "OpticsModes")) apply {
```

Comment (dedmen) [reply addons/arsenal/functions/fnc_statTextStatement_scopeVisionMode.sqf]: Wow I "fixed" from configclasses to ConfigClasses :man_facepalming:

**Comment by dedmen** [comment]
Currently the stat is that the top priority wise. Should it be moved to the bottom of the stack?

**Comment by alganthe** [comment]
would be better at the bottom so more important stats don't overflow to the next page.

**Comment by dedmen** [comment]
Is that correct? @alganthe

**Comment by alganthe** [comment]
yes

**Pull request merged**